src/runtime/block_meta.rs

```diff
@@ -33,17 +33,17 @@ impl BlockMeta {
     }
 }
 
-/// Block metadata buidler
+/// Block metadata builder
 pub struct BlockMetaBuilder {
-    name: String,
+    type_name: String,
```

Comment:
Changed this because I was trying to track `type_name` vs. `instance_name`, and felt this would help others.